src/components/Dialog-Window/Notes/DialogNote.tsx

```diff
@@ -10,14 +10,38 @@ export type NoteProps = {
 export const Note: React.FC<NoteProps> = ({ maxLength, id }) => {
   const [userData, setUserData] = useLocalStorage(id);
   const [note, setNote] = React.useState(userData[id].note ?? "");
+  const [dynamicSavingText, setDynamicSavingText] = React.useState("");
+  const [savingTextTimeout, setSavingTextTimeout] =
+    React.useState<NodeJS.Timeout>();
+
+  // TODO: Translate
+  const savingTextLabel = "lagring...";
+  const savedTextLabel = "lagret";
+
+  const setSavingText = (): void => {
+    setDynamicSavingText(savingTextLabel);
+    setSavingTextTimeout(
+      setTimeout(() => {
```

Comment:
`setTimeout` looks to tsc like it returns a NodeJS.Timeout, but is in fact `window.setTimeout`, which returns a `number`.

This won't change anything, but might clear up the typing for future devs.
 
```suggestion
      window.setTimeout(() => {
```

Reply:
✅